phpstan.neon.dist

```diff
@@ -5,4 +5,6 @@ parameters:
 	level: max
 	paths:
 		- src
-	checkGenericClassInNonGenericObjectType: false
+	ignoreErrors:
+		-
+			identifier: missingType.generics
```

Comment:
cant we add theses errors to the baseline instead? This hides all these errors without the possibility then to fix them

Reply:
We don't want to fix these errors at all. And there are 75 errors, which would make the baseline quite large.

Reply:
I know checked it out and had a look on these errors. I know now what you mean. Ideally we would want to fix them, but it's not so easy and kinda tedious tbh.

Reply:
I didn't add this rule, I just fixed the deprecation. Can we discuss this separately whether it makes sense?

Reply:
Yeah - im fine with that config for now :)